source/route_auth.js

```diff
@@ -12,27 +12,27 @@ angular.module( "vokal.RouteAuth", [ "LocalStorageModule" ] )
         redirectPath = path;
     };
 
-    this.$get = [ "$q", "$location", "$rootScope", "localStorageService",
+    this.$get = [ "$q", "$location", "$rootScope",
 
-        function ( $q, $location, $rootScope, localStorageService )
+        function ( $q, $location, $rootScope )
         {
             var service = {
-
+                storageType: localStorage,
                 loadRoles: function ()
```

Comment:
I think you'd have to check for it anyways since it could be set to `null` by the user